src/mocsolver.jl

```diff
@@ -336,18 +421,23 @@ function add_q_to_φ!(sol::MoCSolution, prob::MoCProblem)
     @inbounds for i in 1:NRegions
         xs = getxs(prob, i)
         @unpack Σt = xs
-
+        if volumes[i] < 1e-12 # deal with zero volume FSRs
+            volumes[i] = 1e-12
+        end
```

Comment:
a volume could be zero because it might not be crossed by any track, right? The following 

```julia
# define a const where in a good location
const MIN_VOLUME = 1e-12 

# ...

volume = volumes[i]
volume = volume < MIN_VOLUME ? MIN_VOLUME : volume
```
then use `volume` in the following code.

Reply:
the OpenMoC you shared with me in the past made the volume equal to 1e30. What is going on? I need to be sure you are not just copying code but you understand what you are copying at least. Please elaborate.

Reply:
Thank you, this is a very helpful suggestion! This made me realize a mistake.  `volume` should be set to 1e30 so that `φ[ig] /= (Σt[g] * volume)` evaluates to near zero. This is necessary to avoid NaN if a volume is evaluated as zero. `volumes[i]` should not be changed. I fixed this.

I also updated this to use `const`, and to use the conditional operator you suggested.